runtime/oti/j9nonbuilder.h

```diff
@@ -4925,6 +4925,8 @@ typedef struct J9VMThread {
 #endif /* J9VM_JIT_TRANSACTION_DIAGNOSTIC_THREAD_BLOCK */
 #if defined(OMR_GC_CONCURRENT_SCAVENGER)
 	struct J9GSParameters gsParameters;
+	void* gsEvacuateBase;
```

Comment:
@amicic Aleks, do you think it's okay to move those fields outside of #if defined(OMR_GC_CONCURRENT_SCAVENGER) in ? We only use them for concurrent scavenge, but the issue Daryl raised is that we can't reference in omr project, where I need to access these fields.

Reply:
Is't the dilemma then if the fields should reside to J9VMThread or OMRVMThread ? This build flag is an OMR build flag, and all JIT code that access those field could/should be under this flag too.